applications/nrf5340_audio/src/bluetooth/bt_management/bt_mgmt.h

```diff
@@ -27,6 +27,9 @@
 	BT_LE_ADV_PARAM(BT_LE_ADV_OPT_CONNECTABLE, BT_GAP_ADV_FAST_INT_MIN_1,                      \
 			BT_GAP_ADV_FAST_INT_MAX_1, NULL)
 
+/* Broadcast name can max be 32 bytes long, so this will be the limit for both */
+#define BLE_SEARCH_NAME_MAX_LEN 32
```

Comment:
Because the PBP says so:
```
5.1.1 Format
The format for the Broadcast_Name AD Type is a UTF-8 encoded string containing a minimum of 4 and a 
maximum of 32 human-readable characters.
```

Reply:
> Because the PBP says so:
> 
> ```
> 5.1.1 Format
> The format for the Broadcast_Name AD Type is a UTF-8 encoded string containing a minimum of 4 and a 
> maximum of 32 human-readable characters.
> ```

Ah, thanks. I was unaware of that. Odd to have a upper layer profile that, rather than the assigned numbers document :D 

But you are also using this to defined `char []` in the code where you have a NULL terminator, right (since you are doing string operations)? 

I guess those arrays should use `BLE_SEARCH_NAME_MAX_LEN + 1` or this should be defined as `33`